### ISSUE TITLE
Test Set Inference

### ISSUE THREAD

**Issue by huyue132** [LezJ/SimMLM#4]
Your work is fascinating! However, I've encountered a problem: how can I use the trained expert and joint models to test local samples and output the relevant evaluation metrics?

**Comment by LezJ** [comment]
Hi, thanks for your interests.
At the end of `pipeline_joint_training.py`, we evaluate on the validation set of BraTS 2018, starting here: https://github.com/LezJ/SimMLM/blob/c9be74ba913db08013b8e573ce6cbda25da9ea1a/pipeline_joint_training.py#L171


It saves the Dice metrics for the three subtasks across all possible complete and missing conditions with the best (the checkpoint saved as `*.bst`) and final-epoch (saved as `*.final`) model, respectively.


You may refer to it and revise for reporting your local samples' results.
